swift-sdk/Internal/AuthManager.swift

```diff
@@ -101,11 +101,11 @@ class AuthManager: IterableInternalAuthManagerProtocol {
         
         storeAuthToken()
         
+        queueAuthTokenExpirationRefresh(authToken)
+        
         if authToken != nil {
             onSuccess?(authToken)
         }
-        
-        queueAuthTokenExpirationRefresh(authToken)
```

Comment:
What was the reason for moving this call?

Reply:
Semantically there should be no difference but it was to follow the same flow that Android does [here](https://github.com/Iterable/iterable-android-sdk/blob/f698238f800c49b97cd40d892807eca031e5e8a3/iterableapi/src/main/java/com/iterable/iterableapi/IterableAuthManager.java#L49).